src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -76,47 +75,55 @@ public Class<? extends Number> getReturnType() {
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if ((mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE) && getExpr().isSingle() && ChangerUtils.acceptsChange(getExpr(), ChangeMode.SET, Location.class))
-			return new Class[] {Number.class};
-		return null;
+		if ((mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE) && getExpr().isSingle()) {
+			if (ChangerUtils.acceptsChange(getExpr(), ChangeMode.SET, Location.class)) {
+				return new Class[] {Number.class};
+			}
+		}
+		return new Class[0];
 	}
 	
 	@Override
-	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) throws UnsupportedOperationException {
-		assert delta != null;
-		final Location l = getExpr().getSingle(e);
-		if (l == null)
+	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) throws UnsupportedOperationException {
+		final Object o = getExpr().getSingle(event);
```

Comment:
_Still unresolved from previous review_ 
```suggestion
		Object obj = getExpr().getSingle(event);
```